Formula/oma.rb

```diff
@@ -15,10 +16,28 @@ class Oma < Formula
     sha256 "57f39d3238c12cbe8534a4f571e52328060b106580cb48c72272492f39e8f029" => :x86_64_linux
   end
 
+  depends_on "libxml2"
+  depends_on "numpy"
   depends_on "python"
 
+  resource "biopython" do
+    url "https://files.pythonhosted.org/packages/33/55/becf2b99556588d22b542f3412990bfc79b674e198d9bc58f7bbc333439e/biopython-1.75.tar.gz"
+    sha256 "5060e4ef29c2bc214749733634051be5b8d11686c6590fa155c3443dcaa89906"
+  end
+
+  resource "lxml" do
+    url "https://files.pythonhosted.org/packages/e4/19/8dfeef50623892577dc05245093e090bb2bab4c8aed5cad5b03208959563/lxml-4.4.2.tar.gz"
+    sha256 "eff69ddbf3ad86375c344339371168640951c302450c5d3e9936e98d6459db06"
+  end
+
   def install
-    system "./install.sh", prefix, share
+    venv = virtualenv_create(libexec, python="python3")
```

Comment:
```suggestion
    venv = virtualenv_create(libexec, "python3")
```

Reply:
thanks @jonchang , helped at least for macos.